### DISCUSSION TITLE
Selenium 4 support

### DISCUSSION THREAD

**Discussion by jirikralik** [appium/appium#15595] [Q&A]
Hello guys, is there any roadmap to support selenium 4? Thank you!

**Comment by KazuCocoa** [comment]
Do you mean clients?

I have two working branches.
Other repositories depend on each client repository.
https://github.com/appium/ruby_lib_core/tree/se-v4
https://github.com/appium/python-client/tree/selenium-4

We do not have any concrete plans for grid 4. We're happy if anyone, who is interested in supporting Grid 4, works with us.
